File: Sort and Search/InsertionSort.py
def insertion_sort(arr):
	for i in range(len(arr)):
		k = i
		while(k > 0 and arr[k] < arr[k-1]):
			arr[k], arr[k-1] = arr[k-1], arr[k]
			k -= 1
	return arr

print(insertion_sort([5, 4, 1, 9, 7, 6]))

File: Sort and Search/InsertionSort.js
function insertionSort(arr){
	for(var i=1; i<arr.length; i++){
		j = i;
		while(arr[j]<arr[j-1]){
			temp = arr[j];
			arr[j] = arr[j-1]
			arr[j-1] = temp;
			j--;
		}
	}
	console.log(arr);
}

insertionSort([5, 3, 1, 8, 9, 6]);